Toshi/Models/Yap.swift

```
@@ -132,6 +132,21 @@ public final class Yap: NSObject, Singleton {
 
         if database == nil {
             CrashlyticsLogger.log("Failed to create user database")
+        } else {
+            addSkipBackupAttributeToDatabasePath()
+        }
+    }
+
+    private func addSkipBackupAttributeToDatabasePath() {
+        var databaseUrl = NSURL.fileURL(withPath: UserDB.dbFilePath)
+
+        do {
+            var resourceValues = URLResourceValues()
+            resourceValues.isExcludedFromBackup = true
```

You could also check the `is ubiquitous` thing, but I think SSK or Yap already do it. 🤔

---

not sure why i might want to check it here

---

You can double-check if you want, since you updated YapDatabase and SSK. I'm quite sure the last version I added removed the db from iCloud. 👍 Can't see why they'd change it.

---

why might i want to check if a file at the given path was synced to icloud ?

---

`if this item is synced to the cloud` .. i need to change this anyway

---

All already being handled https://github.com/toshiapp/toshi-ios-client/pull/570/files#diff-e792199191a879a8db3071bd60c4cf6eR128. But it's possible this is failing silently. Try using a `do/catch` block instead, so we can log if it fails? 👍

---

👍